### DISCUSSION TITLE
See 'Other' files under ISFS?

### DISCUSSION THREAD

**Discussion by CraigRegester** [intersystems-community/vscode-objectscript#870] [Q&A] [answered]
I searched for an answer to this but was unsuccessful - is there a way to see/edit 'Other' files (e.g., .LUT, .HL7, .X12, etc) like we can do with Studio but instead in VS Code with ISFS?

Trying to complete my Teams' transition to using VS Code but stuck only with these items I believe. I can see them on non-ISFS in the ObjectScript Explorer but generally that isn't used with ISFS so wasn't sure of the appropriate workflow with these items.

Generally the GUI is used for these things but occasionally it makes logical sense to edit directly when making significant edits to tables or even schemas.

**Comment by isc-bsaviano** [comment, marked as answer]
@CraigRegester You can see all files by specifying `filter=*` in your `isfs` uri. You can also specify specific file extensions: `filter=*.dfi,*.lut`. More info can be found [here](https://intersystems-community.github.io/vscode-objectscript/serverside/#filters-and-display-options)

**Comment by CraigRegester** [comment]
Perfect, let me try that but note that the link you referenced for more info was my first stop and it does not mention the * by itself as an option necessary to show those files. I should have tried, to be fair, but hoping the documentation can be more explicit in those filter options in the future. Thanks @isc-bsaviano

**Reply by isc-bsaviano** [reply]
I will make a note that we should include more detail there.

**Reply by CraigRegester** [reply]
> I will make a note that we should include more detail there.

Thank you sir - can confirm the * gave us what was desired as well. Appreciate it.
